### ISSUE TITLE
Build - disable loading node_modules deps from cache in PR/release build

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#7355]
Currently we're getting a lot of build failures in core due to this error in the a11y tests, which is resolved by deleting the node_modules deps cache and forcing the PR to do a fresh `yarn install`. As discussed in a working sesh, we're going to disable loading from cache in the PR and release build actions indefinitely. If a need arises (for example, build times prove to be taking way too long), we can look at re-enabling the cache and fixing whatever is going on currently.

```
/home/runner/work/patternfly/patternfly/node_modules/puppeteer-cluster/dist/Cluster.js:119
                throw new Error(`Unable to launch browser, error message: ${err.message}`);
                      ^

Error: Unable to launch browser, error message: Could not find Chrome (ver. 130.0.6723.91). This can occur if either
 1. you did not perform an installation before running the script (e.g. `npx puppeteer browsers install chrome`) or
 2. your cache path is incorrectly configured (which is: /home/runner/.cache/puppeteer).
For (2), check out our guide on configuring puppeteer at https://pptr.dev/guides/configuration.
    at Cluster.<anonymous> (/home/runner/work/patternfly/patternfly/node_modules/puppeteer-cluster/dist/Cluster.js:119:23)
    at Generator.throw (<anonymous>)
    at rejected (/home/runner/work/patternfly/patternfly/node_modules/puppeteer-cluster/dist/Cluster.js:6:65)

Node.js v20.18.2
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```
